cmd/palomad/root.go

```diff
@@ -33,7 +34,7 @@ func NewRootCmd() *cobra.Command {
 	// set Bech32 address configuration
 	params.SetAddressConfig()
 
-	tempApp := palomaapp.New(cosmoslog.NewNopLogger(), db.NewMemDB(), io.MultiWriter(), true, db.OptionsMap{})
+	tempApp := palomaapp.New(cosmoslog.NewNopLogger(), db.NewMemDB(), io.MultiWriter(), true, simtestutil.NewAppOptionsWithFlagHome(palomaapp.DefaultNodeHome))
```

Comment:
Why are you using the `simtestutil` package outside of simulation? Why not pass proper options?

Reply:
@vishal-kanna can we resolve now this so we can merge into the next release, please?

Reply:
Okay @taariq